app/src/main/java/com/github/se/cyrcle/ui/card/CardScreen.kt

```diff
@@ -28,113 +28,31 @@ import androidx.compose.ui.platform.testTag
 import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
-import com.mapbox.geojson.Point
 
-// Function to convert ParkingProtection enum to human-readable string
-fun convertProtectionToString(protection: ParkingProtection): String {
-  return when (protection) {
-    ParkingProtection.INDOOR -> "Indoor protection"
-    ParkingProtection.COVERED -> "Covered protection"
-    ParkingProtection.NONE -> "No protection"
-  }
-}
-
-// Function to convert ParkingRackType enum to human-readable string
-fun convertRackToString(rackType: ParkingRackType): String {
-  return when (rackType) {
-    ParkingRackType.TWO_TIER -> "Two-tier rack"
-    ParkingRackType.U_RACK -> "U-Rack"
-    ParkingRackType.VERTICAL -> "Vertical rack"
-    ParkingRackType.WAVE -> "Wave rack"
-    ParkingRackType.WALL_BUTTERFLY -> "Wall butterfly rack"
-    ParkingRackType.POST_AND_RING -> "Post and ring rack"
-    ParkingRackType.GRID -> "Grid rack"
-    ParkingRackType.OTHER -> "Unidentified Rack"
-  }
-}
-
-// Function to convert ParkingCapacity enum to human-readable string
-fun convertCapacityToString(capacity: ParkingCapacity): String {
-  return when (capacity) {
-    ParkingCapacity.XSMALL -> "Less than 10 spots"
-    ParkingCapacity.SMALL -> "10-25 spots"
-    ParkingCapacity.MEDIUM -> "26-50 spots"
-    ParkingCapacity.LARGE -> "51-100 spots"
-    ParkingCapacity.XLARGE -> "More than 100 spots"
-  }
-}
-
-// Sample parking spot data for demonstration
-val parking1 =
-    Parking(
-        "Test_spot_1",
-        null,
-        null,
-        Location(Point.fromLngLat(6.6, 46.2)),
-        listOf(
-            "https://upload.wikimedia.org/wikipedia/commons/7/78/%22G%C3%A4nsemarkt%22_in_Amance_-_panoramio.jpg",
-            "https://upload.wikimedia.org/wikipedia/commons/7/78/%22G%C3%A4nsemarkt%22_in_Amance_-_panoramio.jpg",
-        ),
-        ParkingCapacity.LARGE,
-        ParkingRackType.TWO_TIER,
-        ParkingProtection.COVERED,
-        0.0,
-        true)
-
-// Another parking spot example
-val parking2 =
-    Parking(
-        "Test_spot_2",
-        null,
-        null,
-        Location(Point.fromLngLat(6.7, 46.3)),
-        listOf(
-            "https://upload.wikimedia.org/wikipedia/commons/6/6b/Bicycle_parking_at_Alewife_station%2C_August_2001.jpg"), // Corrected URL
-        ParkingCapacity.SMALL,
-        ParkingRackType.TWO_TIER,
-        ParkingProtection.COVERED,
-        0.0,
-        true)
-
-// Third parking spot example
-val parking3 =
-    Parking(
-        "Test_spot_3",
-        null,
-        null,
-        Location(Point.fromLngLat(7.1, 47.1)),
-        listOf(
-            "https://upload.wikimedia.org/wikipedia/commons/6/6b/Bicycle_parking_at_Alewife_station%2C_August_2001.jpg"), // Corrected URL
-        ParkingCapacity.LARGE,
-        ParkingRackType.TWO_TIER,
-        ParkingProtection.COVERED,
-        0.0,
-        true)
-
-// Main UI Composable function to display card with parking information
```

Comment:
Thanks for removing all the hard-coded data and the now unneccesary helper functions! The code is now much cleaner which I'm glad to see!